Code/src/main/java/nl/utwente/group10/ui/components/lines/ConnectionLine.java

```diff
@@ -46,8 +48,11 @@ public ConnectionLine() {
 
     /**
      * Sets the start position for this Line object
-     * @param x coordinate
-     * @param y coordinate
+     * 
+     * @param x
+     *            coordinate
+     * @param y
+     *            coordinate
```

Comment:
Could you clarify what you mean by 'correctly formatted'?

I gave it a whirl and I didn't see any difference for the generated JavaDoc. The [standard](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format) gives an example where it goes on one line. I understand that Eclipse autoformats `@param` tags to two lines, but I'm not sure that means it's the only correct way.